torch/distributed/_tensor/debug/visualize_sharding.py

```diff
@@ -153,6 +153,10 @@ def visualize_sharding(dtensor, prompt=""):
 
     # Convert offsets to blocks with row_ranges for tabulate
     blocks = _convert_offset_to_ranges(all_offsets)
-    if device_mesh.get_rank() == 0:
+    local_rank_zero_on_all_dim = all(
+        device_mesh.get_local_rank(mesh_dim=dim) == 0
```

Comment:
I think we can simply do this?
`device_mesh.get_local_rank(mesh_dim=0) == 0`

Reply:
this may not be correct for 2-d mesh. For example, saying the mesh is [[0, 1], [2, 3]]. `get_local_rank(mesh_dim=0)  == 0` is `True` on rank 0 and 1 whose first coordinate is 0.

Reply:
Ohh i see